### DISCUSSION TITLE
use better-sqlite error

### DISCUSSION THREAD

**Discussion by luckyLocas** [alex8088/electron-vite#465] [General]
![image](https://github.com/alex8088/electron-vite/assets/23146290/230b6778-e353-4f3b-a07d-e652ab39c1d1)
![Uploading image.png…]()

![image](https://github.com/alex8088/electron-vite/assets/23146290/785fb0d5-873b-42c0-ab19-794ad1fec86e)
尝试了这种方式并未解决，Attempted this method but did not solve it

**Comment by alex8088** [comment]
- target为electron的不是node，其次需要指定electron的header地址 --dist-url=https://electronjs.org/headers
- 也可以通过electron-builder来重新编译 npx electron-builder install-app-deps

**Reply by luckyLocas** [reply]
解决了，大佬牛逼
